frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -314,6 +314,30 @@ export const getPipelineRunWorkspaces = (
   );
 };
 
+export const calculateRelativeTime = (startTime: string, completionTime?: string) => {
+  const start = new Date(startTime).getTime();
+  const end = completionTime ? new Date(completionTime).getTime() : new Date().getTime();
+  const secondsAgo = (end - start) / 1000;
+  const minutesAgo = secondsAgo / 60;
+  const hoursAgo = minutesAgo / 60;
+
+  if (minutesAgo > 90) {
+    const count = Math.round(hoursAgo);
+    return `about ${count} hours`;
+  }
+  if (minutesAgo > 45) {
```

Comment:
What about the `'about an hour'`?

More than 45m to less than 90m can have `1h 29m 59s` be `'less than a hour'`?